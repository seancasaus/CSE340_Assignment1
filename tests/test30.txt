t1  (((a)*).((b)*)).((c).(c)) , t2  (((a)*)|((b)*)).(((c).(c)).(a)) , t3 ((((a)*)|((b)*))*).((((a)*)|((c)*)).(c))#
" aaaabbbbccaaaccababababcccc "
