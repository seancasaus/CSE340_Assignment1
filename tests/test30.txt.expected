t1 , "aaaabbbbcc"
t2 , "aaacca"
t3 , "babababcccc"
